shiny/ui/_html_deps_external.py

```diff
@@ -17,16 +19,44 @@
 
 
 def bootstrap_deps() -> list[HTMLDependency]:
-    dep = HTMLDependency(
-        name="bootstrap",
+    bootstrap_css = HTMLDependency(
+        name="bootstrap-css",
         version=bootstrap_version,
         source={"package": "shiny", "subdir": "www/shared/bootstrap/"},
-        script={"src": "bootstrap.bundle.min.js"},
         stylesheet={"href": "bootstrap.min.css"},
+    )
+
+    bootstrap_js = HTMLDependency(
+        name="bootstrap-js",
+        version=bootstrap_version,
+        source={"package": "shiny", "subdir": "www/shared/bootstrap/"},
+        script={"src": "bootstrap.bundle.min.js"},
+    )
+
+    bootstrap_meta = HTMLDependency(
+        name="bootstrap-meta",
+        version=bootstrap_version,
+        source={"package": "shiny", "subdir": "www/shared/bootstrap/"},
         meta={"name": "viewport", "content": "width=device-width, initial-scale=1"},
     )
-    deps = [jquery_deps(), dep]
-    return deps
+
+    return [
+        jquery_deps(),
+        bootstrap_meta,
+        bootstrap_js,
+        bootstrap_css,
+    ]
+
+
+def bootstrap_deps_suppress(
+    parts: list[Literal["css", "js", "meta"]]
+) -> list[HTMLDependency]:
+    bs_v_suppressed = str(bootstrap_version) + ".9999"
+
+    return [
+        HTMLDependency(name=f"bootstrap-{part}", version=bs_v_suppressed)
+        for part in parts
+    ]
```

Comment:
This is a small thing, but I think I'd prefer this to be more generic, like `htmltools::supressDependencies()`?

Reply:
Does that not exist in py-htmltools? We could add it there. 

There might still be some value in this helper function, though. This function encapsulates both the Bootstrap version and the naming pattern we use. Also, `suppressDependencies()` uses `version="9999"`, but I'm trying to be more careful here and am only suppressing the version used by Shiny. So we'd either have to update the API of `suppressDependencies()` or we'd still use this version.

OTOH, we could drop this function and other packages could use something like `htmltools.suppress_dependency()` instead. As we can see from the shinyswatch experience though, that increases the strength of coupling between shiny and the external packages. If we change how the Bootstrap dependency is structured, we'd need to change third party code as well. If we use this (not publicly advertised) helper function though, we can change the implementation in Shiny without breaking external code or having to update across several packages at once